cube-builder-aws/requirements.txt

```diff
@@ -5,8 +5,8 @@ Flask-Migrate==2.5.2
 Flask-SQLAlchemy==2.4.1
 GeoAlchemy2==0.6.3
 SQLAlchemy[postgresql]==1.3.13
-git+https://github.com/brazil-data-cube/bdc-db@v0.2.1
-git+https://github.com/brazil-data-cube/stac.py@master
+git+https://github.com/brazil-data-cube/bdc-db@b-0.2
```

Comment:
I don't know exactly who is including the dependency ``dateutil``. I recommend to set it to avoid dependency error on the future. I will put it in ``cube-builder`` local.

Reply:
thanks